app/experimenter/static/rapid/components/experiments/ExperimentDetails.tsx

```diff
@@ -68,6 +68,26 @@ const ExperimentDetails: React.FC = () => {
     );
   }
 
+  let monitoring_url;
+  if (experimentData.monitoring_dashboard_url) {
+    monitoring_url = (
+      <>
+        <h3 className="my-4">Monitoring</h3>
+        <p>
+          The live monitoring dashboard can be found here:
+          <a
+            href={experimentData.monitoring_dashboard_url}
+            rel="noopener noreferrer"
+            target="_blank"
+          >
+            {" "}
+            (link here)
```

Comment:
Are we replacing the `(link here)` with the actual link URL eventually?
This currently reads a bit awkward:

> ### Monitoring
> The live monitoring dashboard can be found here: (link here)

Or if we change it to 

> The live monitoring dashboard can be found &lt;here&gt;"

Or even better is possibly just

> Visit &lt;live monitoring dashboard&gt;.

I'll check the Figma, but wasn't sure if the "link here" was a placeholder for the actual URL in some of the specs.

Reply:
I'm not sure, there is no monitoring link in the figma at all so I think we'll have to get @AnaMedinac to update the mocks and then we can PR the relevant changes in.